R/install.r

```diff
@@ -47,6 +47,7 @@
 #'   It defaults to the option \code{"Ncpus"} or \code{1} if unset.
 #' @param force_deps whether to force installation of dependencies even if their
 #'   SHA1 reference hasn't changed from the currently installed version.
+#' @param metadata metadata to add to \code{DESCRIPTION}.
```

Comment:
Need more details here. What sort of object is it? How does the addition happen?